storage/store.go

```diff
@@ -0,0 +1,39 @@
+package storage
+
+import (
+	"context"
+	"sync"
+)
+
+type Store struct {
```

Comment:
Yes, exactly, that's the idea behind it. However, I have to try this out with the Emulator, because there following events:
- `flow.evm.BlockExecuted`,
- `flow.evm.TransactionExecuted`
are not currently available in neither `testnet` nor `mainnet`.